guide/app_star_catalogue.tex

```diff
@@ -1835,18 +1835,18 @@ \subsection{Cross-Identification Catalog File Format}
 \subsubsection{File Format}
 \label{sec:StarCatalogues:CrossIdentificationData:file}
 
-The file \file{cross-id.dat} contains cross reference data for catalogues HIP, SAO, HD and HR, which stored in the plain text format with tab delimiter.
+The file \file{cross-id.cat} contains cross reference data for catalogues HIP, SAO, HD and HR, which stored in the binary format.
 
 \begin{table}[htb]
 \begin{tabularx}{\textwidth}{l|c|X}\toprule
 \emph{Name} & \emph{Type} & \emph{Description}\\\midrule
-HIP	      & int & HIP or Gaia identifier for the star. Used for reference to the main star catalogue.\\%\midrule
-Component & str & Component of the star in the multiple star system.\\%\midrule
-SAO    	  & int & SAO identifier for the star.\\%\midrule
-HD	      & int & HD identifier for the star.\\%\midrule
-HR	      & int & HR identifier for the star.\\\bottomrule
+HIP	      & unsigned long long & HIP or Gaia identifier for the star. Used for reference to the main star catalogue.\\%\midrule
```

Comment:
This should be a sized-int type, like `uint64_t` or Qt's `quint64`.

Reply:
fixed in #4164